src/components/PeopleFilters/PeopleFilters.tsx

```diff
@@ -0,0 +1,108 @@
+import { ChangeEvent, FC } from 'react';
+import cn from 'classnames';
+
+import { SEX_OPTIONS, CENTURIES } from '../../Constants';
+import { SearchLink } from '../SearcLink';
+
+type Props = {
+  activeSex: string | null;
+  query: string;
+  onQueryChange: (event: ChangeEvent<HTMLInputElement>) => void;
+  onSexChange: (value: string | null) => void;
+  getCenturiesParams: (selectedCentury: string) => string[];
+  centuries: string[];
+};
+
+export const PeopleFilters: FC<Props> = ({
+  activeSex,
+  query,
+  onQueryChange,
+  onSexChange,
+  getCenturiesParams,
+  centuries,
+}) => {
+  return (
+    <nav className="panel">
+      <p className="panel-heading">Filters</p>
+
+      <p className="panel-tabs" data-cy="SexFilter">
+        {SEX_OPTIONS.map(sex => (
+          <SearchLink
+            key={sex.name}
+            params={{ sex: sex.value }}
+            className={cn({
+              'is-active':
+                sex.value === activeSex ||
+                (sex.value === '' && activeSex === null),
```

Comment:
the same syntax(I recommend using it in all such cases)
```suggestion
                (!sex.value.length && !activeSex),
```